tests/shardddl3/run.sh

```diff
@@ -2556,6 +2738,9 @@ function run() {
     DM_RestartMaster
 
     DM_DropAddColumn
+    DM_INIT_SCHEMA
```

Comment:
now shardddl3 will take more than 20 minutes, and it's the slowest test. we might move it to another test.

Reply:
https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/dm_ghpr_test/detail/dm_ghpr_test/8769/pipeline

so if CI get slower, we might can't merge all PR today 😂